### PR TITLE
Feature/Testimonial: Add Testimonial component

### PR DESCRIPTION
**Pull request by AbigailFernandes**

### CODE REVIEW

File: src/components/Testimonial.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import clientAvatar from '../img/client.jpg';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import faQuoteLeft from '@fortawesome/fontawesome-free-solid/faQuoteLeft';
+
+export default ({
+  clientName = 'Juhi Jayaram',
```
Comment (sanchitnevgi) [inline src/components/Testimonial.js]: Just nitpicking, but prefer grouping relevant information in an object. Like `client`. And reference with `client.name` and `client.review`

Comment (AbigailFernandes) [reply src/components/Testimonial.js]: Yeah! Makes sense!

File: src/components/Testimonial.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  clientReview = 'Finding the shampoo that is both healthy and nice for your hair is not easy for any woman… So when I found how many hair care products are offered here I was hooked once and for all!',
+}) => (
+  <section className="section-testimonial">
+    <div className="section section-title">
+      <h3 className="title is-3">Customer Saying</h3>
+    </div>
+    <div className="columns is-mobile">
+      <div className="column is-half is-offset-one-quarter testimonial-item">
+        <div className="margin-btm text-center">
+          <img src={clientAvatar} alt="client avatar" className="img-circle" />
```
Comment (sanchitnevgi) [inline src/components/Testimonial.js]: Use bulma's `image` class over defining custom responsive styles

File: src/components/Testimonial.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  clientName = 'Juhi Jayaram',
+  clientReview = 'Finding the shampoo that is both healthy and nice for your hair is not easy for any woman… So when I found how many hair care products are offered here I was hooked once and for all!',
+}) => (
+  <section className="section-testimonial">
+    <div className="section section-title">
+      <h3 className="title is-3">Customer Saying</h3>
+    </div>
+    <div className="columns is-mobile">
+      <div className="column is-half is-offset-one-quarter testimonial-item">
+        <div className="margin-btm text-center">
```
Comment (sanchitnevgi) [inline src/components/Testimonial.js]: `margin-btm` is ambiguous name. Consider renaming to `margin-btm-2`, etc

File: src/layouts/styles.scss [hunk 4]
```diff
@@ -1,3 +1,4 @@
+@import '../stylesheets/testimonial.scss';
```
Comment (sanchitnevgi) [inline src/layouts/styles.scss]: Add custom styles after font loading

Comment (sanchitnevgi) [reply src/layouts/styles.scss]: Even better, import `testimonial.scss` directly in Testimonial component

File: src/stylesheets/testimonial.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+.text-center {
+    text-align: center;
+}
+
+.margin-btm {
+    margin-bottom: 2rem;
+}
+
+.testimonial-item {
+    margin-bottom: 3rem;
```
Comment (sanchitnevgi) [inline src/stylesheets/testimonial.scss]: You can use an `scss` mixin for margin that takes value as argument

Comment (AbigailFernandes) [reply src/stylesheets/testimonial.scss]: I did that initially but I still would need both these classes. I will do that.

File: src/layouts/styles.scss (hunk 4 above)
Comment (AbigailFernandes) [reply src/layouts/styles.scss]: Alright

File: src/components/Testimonial.js (hunk 3 above)
Comment (AbigailFernandes) [reply src/components/Testimonial.js]: Alright

File: src/components/Testimonial.js (hunk 2 above)
Comment (AbigailFernandes) [reply src/components/Testimonial.js]: Yeah! Will do that.

**Pull request merged**